### PR TITLE
allow to skip management node in portal load-balancing

### PR DESCRIPTION
**Pull request by julsemaan**
# Description
allow to skip management node in portal load-balancing

# Impacts
haproxy-portal

# Delete branch after merge
YES

# NEWS file entries
## Enhancements
* Allow to skip management node in portal load-balancing when running in a cluster

### CODE REVIEW

File: conf/pf.conf.defaults [hunk 1]
```diff
@@ -998,6 +998,14 @@ dns_on_vip_only=disabled
 # Centralize the deauthentication to the management node of the cluster.
 centralized_deauth=enabled
 #
+# active_active.portal_on_management
+#
+# Process captive portal requests on the management server (the current load balancer).
+# Disabling it will make the management server only proxy requests to other servers.
+# Useful if your load balancer cannot handle both tasks.
+# Changing this requires to restart haproxy-portal.
```
Comment (extrafu) [inline conf/pf.conf.defaults]: to restart the haproxy-portal services

all places.

File: html/pfappserver/lib/pfappserver/I18N/en.po [hunk 2]
```diff
@@ -9691,6 +9691,10 @@ msgstr "Total"
 msgid "active_active"
 msgstr "Active Active"
 
+# conf/documentation.conf
+msgid "active_active.portal_on_management"
+msgstr "Captive-portal on management"
```
Comment (extrafu) [inline html/pfappserver/lib/pfappserver/I18N/en.po]: Captive portal

**Comment by julsemaan** [comment]
Done

**Comment by extrafu** [comment]
@fdurand If all looks good to you, please merge ASAP.

**Pull request merged**